src/cli/load.test.js

```diff
@@ -302,7 +308,10 @@ describe("cli/load", () => {
           successCase({
             name: "with a multiple repositories",
             args: ["foo/bar", "bar/zod", "--output", "bar/zod"],
-            loadOptions: {output: barZod, repoIds: [fooBar, barZod]},
+            loadOptions: {
```

Comment:
I'm confused where all these unrelated changes are coming from (prettier doesn't usually churn the codebase).

Reply:
It failed on the prettier check on `yarn test`, this is the result of `yarn prettify`.

Reply:
interesting; on my machine, both before and after were `pretty` enough. we must have slightly different config.